src/Settings/Aviant.SettingsGroup.json

```diff
@@ -53,6 +53,22 @@
     "min":          -128.0,
     "max":          0.0
 },
+{
+    "name":        "preDeliveryBingoLimit",
+    "shortDesc":   "Pre-delivery bingo limit",
+    "type":        "uint32",
```

Comment:
Not important, but these are really floats that just happen to be rounded in practice. Good practice would be to use float here